Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/EventSourced/Migration/AddNewProperty_NoDimensions.feature

```diff
@@ -64,7 +66,7 @@ Feature: Add New Property
           -
             type: 'AddNewProperty'
             settings:
-              newPropertyName: 'aDateOutsideSchema'
```

Comment:
@skurfuerst you had originally introduced this test case with 0f88ed62af06ca3eff45736ae02e53692f3c75a6 but in our opinion node migrations should not be allowed to set properties that are not allowed according to the node type schema. Do you agree to that assumption?